src/app/static-pages/landing-page/sections/social-media-section/social-media-section.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Brauchen wir das spec-File hier? In den anderen Landingpage Komponenten haben wir sie nicht drin.

Reply:
Braucht es nicht zwingend weil die Tests ohnehin nur die Erstellung der Komponente testen.
Schaden tut es aber sicherlich auch nicht.